### DISCUSSION TITLE
Fluxv2 image

### DISCUSSION THREAD

**Discussion by TechGirl-3** [fluxcd/flux2#1047] [Q&A]
Hi,

I wanted to know is there any latest Flux v2 image available on DockerHub for installation of Flux on the EKS cluster. I searched on DockerHub and I could only see Fluxv1 images.

Any suggestion here would be helpful.

Thanks in Advance.

**Comment by stefanprodan** [comment]
Please read the install docs https://toolkit.fluxcd.io/guides/installation/

**Reply by TechGirl-3** [reply]
Can we also manage Flux from AWS  ECR?

**Reply by stefanprodan** [reply]
Reading the docs would answer your question: https://toolkit.fluxcd.io/guides/installation/#air-gapped-environments
